<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureEditor</class>
 <widget class="QWidget" name="TextureEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>512</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutWorkspace">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QFrame" name="frameTarget">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayoutTarget">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameProperties">
       <property name="minimumSize">
        <size>
         <width>261</width>
         <height>480</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>261</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>11</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBoxOrigin">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>241</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>Origin</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QSpinBox" name="spinBoxOriginX">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>29</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>8192</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBoxOriginY">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>59</y>
           <width>51</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>8192</number>
         </property>
        </widget>
        <widget class="QLabel" name="labelOriginX">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>X:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="labelOriginY">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Y:</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>71</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QPushButton" name="buttonOriginLeftCenter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="buttonOriginBottomCenter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="buttonOriginBottomLeft">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonOriginCenter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttonOriginTopCenter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="buttonOriginTopRight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="buttonOriginRightCenter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="buttonOriginTopLeft">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="buttonOriginBottomRight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBoxProperties">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>241</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Properties</string>
        </property>
        <widget class="QCheckBox" name="checkBoxAnimated">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>70</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Animated</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxSmooth">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>70</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Smooth</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxLazy">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>70</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Lazy</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelWidth">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelHeight">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="buttonLoadTexture">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Texture</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/icons/folder_open.png</normaloff>:/icons/folder_open.png</iconset>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditName">
        <property name="geometry">
         <rect>
          <x>52</x>
          <y>10</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonEditJson">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit JSON</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/icons/script.png</normaloff>:/icons/script.png</iconset>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
